src/zope/schema/_bootstrapinterfaces.py

```diff
@@ -45,8 +47,9 @@ def __eq__(self, other):
     __hash__ = zope.interface.Invalid.__hash__  # python3
```

Comment:
(I'm not sure this is correct, but it's not part of _this_ PR.)

Reply:
Yeah, it is probably not correct. It gets us back to #36 and #37. It should probably hash the args to be consistent.